gdxsv/lbs.go

```diff
@@ -378,6 +378,7 @@ func (lbs *Lbs) BroadcastLobbyUserCount(lobby *LbsLobby) {
 			}
 		}
 	}
+	lbs.PublishStatusToDiscord()
```

Comment:
Instead of calling the send function directly, just record that an update is needed.

Reply:
done